### ISSUE TITLE
NullPointerException in response.header(String, int)

### ISSUE THREAD

**Issue by mpkusnierz** [perwendel/spark#1273]
spark.http.matching.ResponseWrapper needs to override this method spark.Response.header(String, int) and call the delegate to add the int Header; otherwise this method throws a NullPointerException

java.lang.NullPointerException: null
	at spark.Response.header(Response.java:166)

**Comment by lepe** [comment]
Yes, I believe you are right. I fixed the code in the [unofficial release-5](https://github.com/Intellisrc/spark), which I'm planning to release within the next week (if all goes smoothly).
